lib/getAssetInfo.ts

```diff
@@ -16,6 +16,22 @@ export async function getFileInfo(id: string) {
   if (directusFile === null) throw new Error("No directusFile found");
   return {
     path: await getAssetPath(directusFile.id),
+    blurDataURL: await getBlurURL(directusFile.id),
     ...directusFile,
   };
 }
+
+async function getBlurURL(id: string) {
+  const path = await getAssetPath(id);
+  return encodeImageAsBase64(`${path}&height=200&width=200&quality=25`);
+}
+
+async function encodeImageAsBase64(url: string) {
```

Comment:
This function does more than just encode the image as base64. It also fetches the image from an url and creates a data url from the base64 encoded content.